TADashboard/src/main/java/page_objects/DataProfilesDisplayFieldsPage.java

```diff
@@ -0,0 +1,23 @@
+package page_objects;
+
+import com.logigear.control.common.imp.CheckBox;
+import com.logigear.control.common.imp.Label;
+import utils.common.Constants;
+
+import java.util.List;
+
+public class DataProfilesDisplayFieldsPage extends BasePage {
+    private Label profileHeader = new Label("css=#profilesettings > tbody > tr > td.profilesettingheader");
+    private Label displayFieldsLabels = new Label("css=#profilesettings > tbody label");
+    private CheckBox dynamicCheckBox = new CheckBox("xpath=//table[@id='profilesettings']/tbody//label[contains(text(),'%s')]/input");
+
+    public List<String> getAllDisplayFieldsLabels() {
+        displayFieldsLabels.waitForDisplay();
+        return displayFieldsLabels.getListText();
+    }
+
+    public boolean isFieldPrefixedWithCheckBox(String labelName) {
+        dynamicCheckBox.setDynamicValue(labelName);
```

Comment:
the way you name this (dynamicCheckbox) doesn't make any relation to FieldPrefixed

Reply:
I have fixed it. Thank you so much